src/World.h

```diff
@@ -451,9 +452,9 @@ class cWorld :
 	Returns the UniqueID of the spawned minecart, or cEntity::INVALID_ID on failure. */
 	UInt32 SpawnMinecart(double a_X, double a_Y, double a_Z, int a_MinecartType, const cItem & a_Content = cItem(), int a_BlockHeight = 1);
 
-	/** Spawns a boat at the given coordinates.
+	/** Spawns a boat at the given coordinates. The type is the wood type of the boat.
 	Returns the UniqueID of the spawned boat, or cEntity::INVALID_ID on failure. */
-	UInt32 SpawnBoat(double a_X, double a_Y, double a_Z);
+	UInt32 SpawnBoat(double a_X, double a_Y, double a_Z, cBoat::eBoatType a_BoatType = cBoat::btOak);
```

Comment:
I think the param should not have a default, to force each usage to strictly take care of the type.